### PR TITLE
product-landing-page

### PR DESCRIPTION
**Pull request by Nooria150**
1. I cloned the repository to the local machine.
2. created feature branch.
3. added linters.
4. created the html and css files.
5. started coding in html file.
6. after that I did css codes.
7. I passed it in freecodcamp.
8. added readme.md.

### CODE REVIEW

File: styles.css
Comment (ahzamir) [inline styles.css]: - **[OPTIONAL]** - Give your Header a Background Color and also a z-index to make it show on other elements.

File: index.html
Comment (ahzamir) [inline index.html]: - This code is not aligned correctly remove 2 spaces.

File: index.html
Comment (ahzamir) [inline index.html]: - I can not see the opening tag for these closing div tags so remove them.

**Review by ahzamir — CHANGES_REQUESTED** [review]
### Hi @Nooria150 🙋‍♀️,

> Excellent work! You have done a great job implementing the project requirements but your project still misses some of the requirements:

- Although you have done a great job by implementing some of the requirements, but still you need to work on your code in order to get the final approval.
  - Check the comments bellow for more details.
# Status: Required Changes 🔁
Make changes to your code and commit and request for a re-review.

AlmostThereReachGIF

 ## Highlights 🌞
 
🎯 Great Design implemented.
image

✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed and completed the Product Landing Page and all requirements.
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
❌Your code is clean
✔️Your project has a professional README.md

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by Nooria150** [comment]
I Implement the feedback. #ahzamir

**Comment by Nooria150** [comment]
I implement the feedback. @ahzamir

**Review by ahzamir — APPROVED** [review]
### Hi @Nooria150 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
image

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

GreatJobThumbsUpGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Product Landing Page project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your project has a professional README.md file

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by ahzamir** [comment]
@Nooria150 

- Why don't you merge your pull-request after getting the approval?

**Comment by Nooria150** [comment]
sorry sir! I wont this mistake.

**Pull request merged**
